plugins/periskop/README.md

```diff
@@ -0,0 +1,67 @@
+# periskop
+
+[Periskop](https://periskop.io/) is a pull-based, language agnostic exception aggregator for microservice environments.
+
+![periskop-logo](https://i.imgur.com/z8BLePO.png)
+
+## Periskop aggregated errors
+
+The Periskop Backstage Plugin exposes a component named `PeriskopErrorsTable`.
+Each of the entries in the table will direct you to the error details in your deployed Periskop instance location.
+
+![periskop-errors-card](./docs/periskop-plugin-screenshot.png)
+
+## Setup
+
+1. Configure the [periskop backend plugin](https://github.com/backstage/backstage/tree/master/plugins/periskop-backend/)
+
+2. If you have standalone app (you didn't clone this repo), then do
```

Comment:
I think it's best to keep the instructions from the point of view of a standalone app, that's when you'd be reading this